config/interceptor/e2e-test/otel/deployment.yaml

```diff
@@ -21,3 +21,11 @@ spec:
           value: "true"
         - name: KEDA_HTTP_OTEL_METRIC_EXPORT_INTERVAL
           value: "1"
+        - name: KEDA_HTTP_TRACING_ENABLED
+          value: "true"
+        - name: KEDA_HTTP_TRACE_EXPORTER
+          value: otlphttp
+        - name: KEDA_HTTP_TRACE_ENDPOINT
+          value: "opentelemetry-collector.open-telemetry-system:4318"
+        - name: KEDA_HTTP_TRACE_INSECURE
+          value: "true"
```

Comment:
I didn't noticed it with the first PR was created, but we should respect the OTEL spec for env vars: https://opentelemetry.io/docs/specs/otel/protocol/exporter/

Could you change the new envs to respect (or use) the SDK envs?

Reply:
refactored to using the OTEL SDK envs instead